airflow/models.py

```diff
@@ -1460,6 +1460,15 @@ def __init__(
             logging.warning(
                 "start_date for {} isn't datetime.datetime".format(self))
         self.end_date = end_date
+        # could add "not callable(attr) "
+        all_triggers = [getattr(TriggerRule, attr) \
```

Comment:
Can we add a more explicite `is_valid(trigger_rule)` `@classmethod` or an `all_trigger_rules` `@classmethod` to `TriggerRule` instead?